test/powershell/Language/Classes/Scripting.Classes.Modules.Tests.ps1

```diff
@@ -71,3 +71,41 @@ Import-Module Random
     }
 
 }
+
+Describe 'Module reloading with Class definition' -Tags "CI" {
+
+    BeforeAll {
+        Set-Content -Path TestDrive:\TestModule.psm1 -Value @'

```

Comment:
I just tried using `New-Module`, and it doesn't work as expected -- `Get-PassedArgsRoot` doesn't return anything, and PSv5.1 has the same behavior. Maybe there is a bug somewhere ☹️ 

```
PS D:\> New-Module -Name ReloadingTest -ScriptBlock {
>> $passedArgs = $args
>> class Root { $passedIn = $passedArgs }
>> function Get-PassedArgsRoot { [Root]::new().passedIn }
>> function Get-PassedArgsNoRoot { $passedArgs }
>> } -ArgumentList 'abc'

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     0.0        ReloadingTest                       {Get-PassedArgsNoRoot, Get-PassedArgsRoot}

PS D:\>
PS D:\> Get-PassedArgsRoot
PS D:\>
PS D:\> Get-PassedArgsRoot
PS D:\>
PS D:\> Get-PassedArgsNoRoot
abc
PS D:\>
```